app/src/main/java/com/amaze/filemanager/asynchronous/asynctasks/LoadFilesListTask.java

```diff
@@ -118,7 +119,9 @@ public LoadFilesListTask(
         if (hFile == null) {
           hFile = new HybridFile(OpenMode.SMB, path);
         }
-
+        if (!hFile.getPath().endsWith("/")) {
+          hFile.setPath(hFile.getPath() + "/");
```

Comment:
Why is this needed?

Reply:
Need to append a training slash for SMB file loading if one is not present at the end of path. I had raised this point on telegram. Logic from AppBar button presses doesn't have one at the end.

Reply:
Shouldn't it be done before in the execution?

Reply:
We only have open mode switch in this asynctask, before this we don't modify path based on open modes. 
Plus it's not good idea to write such logic in Fragment side.